.circleci/config.yml

```diff
@@ -7,7 +7,13 @@ version: 2.1
 executors:
   # CircleCI base Node + Headless browsers + Clojure CLI - big one
   # Maildev runs by default with all Cypress tests
-  clojure-and-node-and-browsers:
+
+  builder:
+    working_directory: /home/circleci/metabase/metabase/
+    docker:
+      - image: metabase/ci:circleci-java-11-clj-1.10.3.929-07-27-2021-node-browsers
+
+  tester:
     working_directory: /home/circleci/metabase/metabase/
     docker:
       - image: metabase/ci:circleci-java-11-clj-1.10.3.929-07-27-2021-node-browsers
```

Comment:
Aren't these exactly the same?

Reply:
yes, in this state of this PR they're the same, but the difference is that the tester spins up a few more services that are not needed for most of the pipeline. What I wanted to do is to use a builder runner to use it through the whole pipeline and then only start maildev, postgres and mysql in the e2e tests tier.